Decisions.MongoDB/FetchDocumentsStep.cs

```diff
@@ -60,6 +61,44 @@ public MongoDBFilter[] Filters
             }
         }
 
+        [WritableValue]
+        private MongoDBSort[] sortFields;
+
+        [PropertyClassification(5, "Sort By Fields", SETTINGS_CATEGORY)]
+        [ArrayTypeEditor(false, false, false, false, ArraySortOrder.None)]
+        public MongoDBSort[] SortFields
+        {
+            get
+            {
+                UpdateSortFieldsInArray(sortFields);
+                return sortFields;
+            }
+            set
+            {
+                sortFields = value;
+                UpdateSortFieldsInArray(sortFields);
+                OnPropertyChanged();
```

Comment:
Do we need to fire an InputData changed event here as well?

Reply:
Good question, but no; unlike the filters, the sort fields don't require any inputs, so InputData doesn't depend on this property.